lib/screen/book_editor.dart

```diff
@@ -103,8 +105,13 @@ class _BookEditorState extends State<BookEditor> {
       });
     }
     _book.addBook(uid);
-    Navigator.pop(context);
-    Navigator.of(context).pushReplacement(MaterialPageRoute(builder: (context) => HomeScreen()));
+    Navigator.pushAndRemoveUntil(
```

Comment:
What's the difference between `pusReplacement` and `pushAndRemoveUntil` ?

Reply:
pushReplacement redirect and put Home at the top of the widget stack but we can still do back, in fact when I am using it I am getting the back button on the Home Screen. But pushRemoveUntil clean the stack and keep only the HomeScreen.